components/UploadForm.vue

```diff
@@ -230,7 +218,11 @@ export default class UploadForm extends Vue {
         // eslint-disable-next-line prefer-destructuring
         this.fileBlob = files[0]
         if (this.isImage) {
-          this.exifInfo = await exifr.parse(files[0])
+          try {
+            this.exifInfo = await exifr.parse(files[0])
+          } catch (exifInfo) {
+            this.exifInfo = null
```

Comment:
```suggestion
          } catch (err) {
            console.error(err)
            this.exifInfo = null
```
I think it is better to log the error just in case